.github/workflows/eks-deploy.yml

```diff
@@ -52,13 +54,18 @@ jobs:
 
       - name: Deploy to EKS
         run: |
-          curl -o kubectl https://amazon-eks.s3.us-west-2.amazonaws.com/1.20.4/2021-04-12/bin/linux/amd64/kubectl
+          # install kubectl
+          curl -o kubectl https://s3.us-west-2.amazonaws.com/amazon-eks/1.20.4/2021-04-12/bin/darwin/amd64/kubectl
           chmod +x ./kubectl
-          mkdir -p $HOME/bin && cp ./kubectl $HOME/bin/kubectl && export PATH=$PATH:$HOME/bin
-          curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
-          sudo mv /tmp/eksctl /usr/local/bin
+          mkdir -p $HOME/bin && mv ./kubectl $HOME/bin/kubectl && export PATH=$PATH:$HOME/bin
+
+          # set and use kube context
           aws eks --region us-east-1 update-kubeconfig --name ${{ inputs.cluster_env }}-cluster
 
-          sed -i "s/${{ inputs.service_name }}:${{ inputs.env }}/${{ inputs.service_name }}:${{ inputs.version }}/g" config/${{ inputs.service_name }}/eks/eks-${{ inputs.env }}-deployment.yaml
-          kubectl apply -f config/${{ inputs.service_name }}/eks/eks-${{ inputs.env }}-deployment.yaml
-          kubectl rollout restart deployment ${{ inputs.service_name }} -n littera-services-${{ inputs.env }}
\ No newline at end of file
+          # replace vars in eks manifest file
+          sed -i "s/${{ inputs.service_name }}:${{ inputs.cluster_env }}/${{ inputs.service_name }}:${{ inputs.version }}/g" config/${{ inputs.service_name }}/eks/eks-${{ inputs.cluster_env }}-deployment.yaml
```

Comment:
this was changed from `inputs.env` to `inputs.cluster_env` - it needs to stay as `inputs.env` (ie: dev vs testing in the dev cluster)

Reply:
resolved.